### ISSUE TITLE
Social Links has Leading /

### ISSUE THREAD

**Issue by emptymalei** [tract-docs/tract#13]
> I originally posted this in one of the commits. I think it is better to post it as an issue.

Hi @lucperkins , 

Thanks for this excellent theme. 👍 

I am running into a strange bug about social links.

The social link becomes something like this

```
<a class="button is-normal is-black" href="/https:/github.com/emptymalei/theflow" target="_blank">
```

This `/` in front of the github link is strange.

Then I noticed this:

https://github.com/tract-docs/tract/blob/6b9d16634f23fa8b43be83c3d311829d71bcc178/layouts/partials/components/button.html#L4-L7

I am thinking, is it due to this `relLangURL`? I am reading through this part. It seems that the relLangURL also appends to URLs start with http. At least this explains the behaviour I saw.

I did a test by changing the part to the following and it worked. However, I am not so proficient in hugo so I am posting it here for reference.

```
{{ $url := .url }}
{{ $isExternal := hasPrefix .url "http" }}
{{ if $isExternal }}
{{ $url := $url | relLangURL }}
{{ end }}
```

Thank you!

**Comment by lucperkins** [comment]
@emptymalei Are you sure that you're not using the wrong URL in your config? It looks like the URL you're using is `/https:/github.com/emptymalei/theflow`.

**Comment by emptymalei** [comment]
I am using `emptymalei/theflow` in the config.

https://github.com/emptymalei/theflow/blob/55463baf358f1ee7039f1892fce37012ae496997/config.yaml#L24-L28

The version of Hugo is 0.69.2
https://github.com/emptymalei/theflow/blob/master/.github/workflows/deploy.yml#L20

**Comment by lucperkins** [comment]
@emptymalei I cloned your repo and ran the site locally and it seems to be working just fine. Hmmm. Not quite sure what's up here.

**Comment by emptymalei** [comment]
What I am experiencing is that if I serve it using `hugo server`, everything is fine. However, whenever I build the site using `hugo`, I see this problem of leading `/`.

Maybe it is a hugo bug. I'll report there.

**Comment by lucperkins** [comment]
@emptymalei I fixed this in acd6a5e2df58e3c527d96f671b50ef6faed128ac. Update your submodule and you should be ready to go.

**Comment by emptymalei** [comment]
Thank you, @lucperkins !
